### PR TITLE
docs(as/cbr/cse): processing Fields Referencing Huawei Cloud

### PR DESCRIPTION
**Pull request by wangzepeng666**
# 1、resrouce_flexibleengine_cbr_vault
### 1.1 script test
image
image
image

# 2、resrouce_flexibleengine_cse_microservice_engine
### 2.1 script test
image
image
image

### 2.2 unit test already has the enterprise_project_id field.

# 3、data_source_flexibleengine_as_groups
### 3.1 unit test
```hcl
=== RUN TestAccDataSourceASGroup_basic
=== PAUSE TestAccDataSourceASGroup_basic
=== CONT TestAccDataSourceASGroup_basic
--- PASS: TestAccDataSourceASGroup_basic (191.11s)
PASS

coverage: 3.7% of statements in ../../../terraform-provider-flexibleengine/...
```

### 3.2 script test
image
image
image

### CODE REVIEW

File: flexibleengine/acceptance/data_source_flexibleengine_as_groups_test.go [hunk 1]
```diff
@@ -42,3 +48,12 @@ data "flexibleengine_as_groups" "groups" {
 }
 `, testASGroup_basic(name))
 }
+
+func testAccDataSourceASGroup_enterpriseProjectId(name string) string {
+	return fmt.Sprintf(`
+%s
+data "flexibleengine_as_groups" "groups" {
+	enterprise_project_id = 0
```
Comment (chengxiangdong) [inline flexibleengine/acceptance/data_source_flexibleengine_as_groups_test.go:56]: format error

File: docs/data-sources/as_groups.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ subcategory: "Auto Scaling (AS)"
 Use this data source to get a list of AS groups.
 
 ```hcl
+variable "ep_id" {}
+
 data "flexibleengine_as_groups" "groups" {
+   enterprise_project_id = ep_id
```
Comment (chengxiangdong) [inline docs/data-sources/as_groups.md:13]:
```suggestion
   enterprise_project_id = var..ep_id
```

File: docs/data-sources/as_groups.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ subcategory: "Auto Scaling (AS)"
 Use this data source to get a list of AS groups.
 
 ```hcl
+variable "ep_id" {}
+
 data "flexibleengine_as_groups" "groups" {
+   enterprise_project_id = var.ep_id
```
Comment (ShiChangkuo) [inline docs/data-sources/as_groups.md:13]: please delete this query parameter beacuse it should add whitelist for most users.

File: docs/data-sources/as_groups.md [hunk 4]
```diff
@@ -30,6 +33,9 @@ The following arguments are supported:
   - **DELETING**: indicates that the AS group is being deleted.
   - **FREEZED**: indicates that the AS group has been frozen.
 
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the AS group.
```
Comment (ShiChangkuo) [inline docs/data-sources/as_groups.md:36]: data source should be without **ForceNew**

**Pull request merged**